### PR TITLE
some IAM doc updates

### PR DESCRIPTION
**Pull request by sbenedicadb**
- troubleshooting guide 
- animation override on url handling

### CODE REVIEW

**Review by rymorale — APPROVED** [review]
Looks good to me 👍

File: beta/iam/guides/validate-messages-in-griffon.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+- In the **Message Behavior** window - review message behavior, including its supported gestures and animations
+- In the **Message Behavior** window - review message size and positioning properties
+- Clicking the **Simulate on Device** button - trigger the currently selected message, causing it to be displayed on the connected client
+
+    ![Inspecting a downloaded message](./../../../.gitbook/assets/iam_simulation.png)
+
+## FAQs
+
+**Q:** What do I do when one of the required extensions is missing? <br />
+**A:** Ensure that each required extension is linked to your project and registered by `MobileCore`. Ref: [registerExtensions](./../../../foundation-extensions/mobile-core/mobile-core-api-reference.md#registerextensions)
```
Comment (calebk1) [inline beta/iam/guides/validate-messages-in-griffon.md:120]: "For more information, please read the [Mobile Core API reference]()".

File: beta/iam/guides/validate-messages-in-griffon.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+**Q:** Why can't I find an event named `Edge Optimize Personalization Request`? <br />
+**A:** Check to ensure each required extensions is on a version that meets its minimum requirement.
```
Comment (calebk1) [inline beta/iam/guides/validate-messages-in-griffon.md:123]: "Ensure that each of the required extensions is at the minimum required version."

**Review by calebk1 — COMMENTED** [review]
A few comments.

Additionally, for our screenshots, we generally cover up the IMS org by painting it over with the black in the header. It's not _explicitly_ required for now, but just something that you should be aware of!

(The minor pet peeve for me is that the notifications bell isn't cleared as well, but that's also not major.)

File: beta/iam/guides/validate-messages-in-griffon.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Ensure that your app has registered all necessary AEP SDK extensions by doing the following:
+
+1. Launch your application with an **AEPAssurance** session active
+
+1. In the Assurance UI, click on **Shared States** in the left-rail navigation
+
+1. Click the **+** button next to the row with a **State Path** of **com.adobe.module.eventhub**
+
+1. Open the **extensions** object, and validate that each of the required extensions exist, and meet the minimum version requirements. The table below shows the minimum versions required for in-app messaging dependencies:
```
[the remark this answers is not in the document]
Comment (sbenedicadb) [reply beta/iam/guides/validate-messages-in-griffon.md:26]: so remove the line about what the table shows?

[the remark this answers is not in the document]
Comment (calebk1) [reply beta/iam/guides/validate-messages-in-griffon.md:26]: Oh, you can keep that line there. I was just fixing up the first sentence.

**Pull request merged**